src/views/statutory/Stats.vue

```diff
@@ -293,12 +293,26 @@ export default {
       }
     },
     byQuorumData () {
-      const present = this.stats.by_body.length * 100 / this.bodies.length
+      const nonQuorum = []
+      for (const body of this.bodies) {
+        if (!body.name.includes('AEGEE', 0)) {
```

Comment:
`if (['antenna', 'contact antenna', 'contact'].includes(body.name)) {`, this is more readable